File: Examples/CSharp/ApplyLicense/ApplyLicenseUsingFileStream.cs
using Aspose.Note;
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;

namespace Aspose.Note.Examples.CSharp.ApplyLicense
{
    public class ApplyLicenseUsingFileStream
    {
        public static void Run()
        {
            // ExStart:ApplyLicenseUsingFileStream
            // ExFor:License
            // ExFor:License.SetLicense(System.IO.Stream)
            // ExSummary:Shows how to load a license for Aspose.Note from a stream.

            Aspose.Note.License license = new Aspose.Note.License();
            using (FileStream myStream = new FileStream("Aspose.Note.lic", FileMode.Open))
            {
                license.SetLicense(myStream);
            }

            // ExEnd:ApplyLicenseUsingFileStream
        }
    }
}

File: Examples/VisualBasic/ApplyLicense/ApplyLicenseUsingFileStream.vb
Imports Aspose.Note
Imports System.Collections.Generic
Imports System.IO
Imports System.Linq
Imports System.Text

Namespace ApplyLicense
    Public Class ApplyLicenseUsingFileStream
        Public Shared Sub Run()
            ' ExStart:ApplyLicenseUsingFileStream
            Dim license As New Aspose.Note.License()
            Dim myStream As New FileStream("Aspose.Note.lic", FileMode.Open)
            license.SetLicense(myStream)
            ' ExEnd:ApplyLicenseUsingFileStream
        End Sub
    End Class
End Namespace
